egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -37,8 +37,9 @@ EOF
 fi
 
 dir=$dir${affix:+_$affix}
+ali_set=train_nodup
```

Comment:
ok but can we call this build_tree_train_set instead?  ali_set is not very clear.